deploy_hpo.sh

```diff
@@ -65,6 +66,11 @@ do
 	esac
 done
 
+resolve_container_runtime
+
+echo "Deploying with runtime: ${CONTAINER_RUNTIME}"
+
+
 # Call the proper setup function based on the cluster_type
 if [ ${setup} == 1 ]; then
```

Comment:
We need to set defaults for setup & cluster_type

Reply:
Added default values as 1 and native for seup and clusterType respectively